notification-hubs-test-app-refresh/src/main/java/com/example/notification_hubs_test_app_refresh/cookbook/DevicePropertyProvider.java

```diff
@@ -0,0 +1,101 @@
+package com.example.notification_hubs_test_app_refresh.cookbook;
+
+import android.app.Activity;
+import android.content.Context;
+import android.graphics.Point;
+import android.os.Build;
+import android.telephony.TelephonyManager;
+import android.view.Display;
+import android.view.Surface;
+import android.view.WindowManager;
+
+import java.util.Locale;
+
+/**
+ * Examples that are ready for copy/paste usage of how to get particular data about the device this
+ * application is running on.
+ */
+public class DevicePropertyProvider {
+    /**
+     * Returns a tag that is normalized across platforms, to allow targeting of any device in a
+     * particular country.
+     *
+     * @return A string that can be used to target devices in a country.
+     */
+    public static String getCountryTag() {
+        return "Country_" + Locale.getDefault().getCountry();
+    }
+
+    /**
+     * Returns a tag that is normalized across platforms, to allow targeting of any device where the
+     * user locale is set to a particular language.
+     * @return A string that can be used to target device using a language.
+     */
+    public static String getLanguageTag() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            return "Language_" + Locale.getDefault().toLanguageTag();
+        }
+
+        // Do you need this value to be populated in Android versions before Lollipop? Please tell us at:
+        // https://github.com/Azure/azure-notificationhubs-android/issues
```

Comment:
If folks are okay with this time trade-off, I'll add an issue to the GitHub repository and link to it specifically.